guids-generator.html

```diff
@@ -456,17 +499,42 @@
                         />
                     </Grid>
                     <Grid item>
-                        <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
+                      <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
                     </Grid>
                 </Grid>
                 { Object.keys(bhashes).length > 0 &&
-                    <List>
-                        { validatedData.map( item =>
-                            <ListItem className={classes.listItem}>
-                                <ListItemText primary={bhashes[item.index].value} style={{color: bhashes[item.index].isError ? 'red' : 'black'}} />
-                            </ListItem>
-                        )}
-                    </List>
+                    <TableContainer component={Paper}>
+                        <Table className={classes.table} size="small">
+                            <TableHead>
+                              <TableRow>
+                                <StyledTableCell>Health card #</StyledTableCell>
+                                <StyledTableCell>Province</StyledTableCell>
+                                <StyledTableCell>Date of birth</StyledTableCell>
+                                <StyledTableCell><span>GUID</span>
+                                  <CopyToClipboard text={validatedData.map(item => item.isError ? '' : bhashes[item.index].value).join('\n\r')} onCopy={() => onCopyButtonClick()}>
```

Comment:
shouldn't it be `\r\n` instead of `\n\r`?
```suggestion
                                  <CopyToClipboard text={validatedData.map(item => item.isError ? '' : bhashes[item.index].value).join('\r\n')} onCopy={() => onCopyButtonClick()}>
```

Reply:
Not working in any combinations. The code has to be changed more broadly.